src/dbnode/persist/fs/index_write.go

```diff
@@ -116,6 +119,7 @@ func (w *indexWriter) Open(opts IndexWriterOpenOptions) error {
 	w.volumeIndex = opts.Identifier.VolumeIndex
 	w.shards = opts.Shards
 	w.snapshotTime = opts.Snapshot.SnapshotTime
+	w.indexVolumeType = opts.IndexVolumeType
```

Comment:
Perhaps if `opts.IndexVolumeType == ""` then should set it to the default constant? Just since it looks like we set the string ref to non-nil value always (so would be strange to have a non-nil string ref with an empty string value `""` in the proto message).